apricot/doc/apricot_http_service.md

```diff
@@ -0,0 +1,23 @@
+## apricot HTTP service
+
+Web server component that implements the Trivial Configuration Endpoint (Apricot TCE). 
+
+It serves JSON and/or plain text structures in order to make essential cluster information available to scripts and other consumers for which the gRPC interface is impractical.
+
+It is strictly **read-only** and only ever responds to `GET`.
+
+### Configuration
+
+To use this feature, the `o2-apricot` running machine has to communicate with Consul via the `--backendUri` option (see [here](apricot.md)).
+
+### Usage and options
+
+To retrieve the information needed regarding FLPs, use the following urls in a web browser.
+
+To retrieve as plain text:
+* `your_machine/inventory/flps` or `your_machine/inventory/flps/text`
```

Comment:
1. Is `your_machine` the name of the machine running `o2-apricot` ? If so, please rename it to something like `<apricot-server>`. 
2. Please add full URLs, e.g. `http://<apricot-server>/inventory/...`
3. Some examples start with `your_machine`, others with `/inventory`. Is this expected ?
4. Could you add a concrete example when running from console with e.g. `curl` or `wget` ?